trk/src/Screens/ClimbList/index.js

```diff
@@ -7,21 +7,30 @@ import writeSignature from "../../NfcUtils/writeSignature";
 import ensurePasswordProtection from "../../NfcUtils/ensurePasswordProtection";
 import androidPromptRef from "../../Components/AndroidPrompt";
 import ClimbsApi from "../../api/ClimbsApi";
+import { AuthContext } from '../../Utils/AuthContext';
+
 
 const ClimbInputData = () => {
+
+  const { currentUser } = useContext(AuthContext);
+  const setter = currentUser;
+
+
+  const { addClimb } = ClimbsApi();
+
   const [name, setName] = useState("");
   const [grade, setGrade] = useState("");
   const [location, setLocation] = useState("");
   const [image, setImage] = useState("");
 
-  const { addClimb, getClimb } = ClimbsApi();
 
   function handleAddClimb() {
     const climb = {
       name,
       grade,
       location,
       image,
+      setter: setter.uid
```

Comment:
'setter' = a whole of info on the user. You have to select the .uid extension to get the user ID